embedded/src/main/java/org/wildfly/core/embedded/EmbeddedHostControllerFactory.java

```diff
@@ -444,8 +426,8 @@ private void exit() {
         private static HostControllerEnvironment createHostControllerEnvironment(File jbossHome, String[] cmdargs, String hostName, long startTime) {
             try {
                 // for SecurityActions.getSystemProperty("jboss.home.dir") in InstallationManagerService
-                System.setProperty("jboss.home.dir", jbossHome.getAbsolutePath());
-                WildFlySecurityManager.setPropertyPrivileged("jboss.home.dir", jbossHome.getAbsolutePath());
+                System.setProperty(HostControllerEnvironment.HOME_DIR, jbossHome.getAbsolutePath());
+                WildFlySecurityManager.setPropertyPrivileged(HostControllerEnvironment.HOME_DIR, jbossHome.getAbsolutePath());
```

Comment:
Not fully reviewing this, but just caught this in passing. We should likely remove the `System.setProperty()` version if we need to do it privileged.